app/datasources/api.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+from fastapi import APIRouter, Depends
+from fastapi import status as http_status
+from starlette.responses import JSONResponse
+
+from app.core.auth_jwt_bearer import JWTBearer
+from app.datasources.model import CreateDatasource, DatasourceResponse
+from app.optscale_api.orgs_api import OptScaleOrgAPI
+
+logger = logging.getLogger("api.organizations")
+router = APIRouter()
+
+
+@router.post(
+    path="",
+    status_code=http_status.HTTP_201_CREATED,
+    response_model=DatasourceResponse,
+    dependencies=[Depends(JWTBearer())]
+)
+async def create_datasource(
+        data: CreateDatasource,
+        org_api: OptScaleOrgAPI = Depends()
+):
+    """
+    Require user authentication token
+
+    Create a Datasource Cloud Account
+    """
+    return JSONResponse(status_code=201,
+                        content={})
```

Comment:
Looks like the formatter didn't run on this file but the pipeline succeeded. Can you check the github action, I suspect it's only running the ruff linter but not the formatter check, it's super easy to miss. I had to do the same in the other repo:  https://github.com/softwareone-platform/mpt-finops-operations/pull/3/files#diff-308ebf025a49dd39145b197e39a2045e1432d02f46b02d59229856e039a0fbaeR37-R39